iteration.qmd

```diff
@@ -669,7 +669,7 @@ If your input data files change over time, you might consider learning a tool li
 
 ### Many simple iterations
 
-Here we've just loaded the data directly from disk, and were lucky enough to get a tidy dataset.
+Here we've just loaded the data directly from disk, and we're lucky enough to get a tidy dataset.
```

Comment:
I think "were" is correct here?

Reply:
Alright, I understood it in a different way. Thanks!